packages/website/src/building-blocs/PageHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import {GithubButton} from './GithubButton';
+import {Tile, TileProps} from './Tile';
+
+export interface PageHeaderProps {
+    title: string;
+    thumbnail?: TileProps['thumbnail'];
+    description?: React.ReactNode;
+    section: string;
+    /**
+     * Path to the component's source file from /packages/react/src/components
+     *
+     * @example '/button/Button.tsx'
+     */
+    componentSourcePath: string;
+}
+
+export const PageHeader: React.FunctionComponent<PageHeaderProps> = ({
+    title,
+    description,
+    thumbnail,
+    section,
+    componentSourcePath,
+}) => (
+    <div className="plasma-page-header">
+        <h2 className="h5-subdued normal-white-space">{section}</h2>
+        <h1 className="h1-light normal-white-space" data-coveo-field="title">
+            {title}
+        </h1>
+        <h3 className="h4-book-subdued" data-coveo-field="description">
+            {description}
+        </h3>
```

Comment:
There are small issues with the title architecture

**Tab Implementation**
![image](https://user-images.githubusercontent.com/966550/155226710-e372807f-1e64-48cf-96a2-2fe95a88261d.png)

**Tab Guidelines**
![image](https://user-images.githubusercontent.com/966550/155226974-849a665b-1f30-43b1-be01-813f1f91fba0.png)

Reply:
What tool do you use to view those? 👀

Reply:
The same problem exists on master though, so out of scope for this PR

Reply:
See this chrome plugin https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en

Out of scope ok, but would it be possible for you to write a stain to fix this?